test/test_package.cpp

```diff
@@ -1436,3 +1436,103 @@ TEST(DDPackageTest, expectationValueExceptions) {
 
     EXPECT_ANY_THROW(dd->expectationValue(xGate, zeroState));
 }
+
+TEST(DDPackageTest, DDFromSingleQubitMatrix) {
+    auto row0        = dd::CVec{0, 1};
+    auto row1        = dd::CVec{2, 3};
+    auto inputMatrix = dd::CMat{row0, row1};
+
+    const dd::QubitCount nrQubits = 1;
+    auto                 dd       = std::make_unique<dd::Package<>>(nrQubits);
+    auto                 matDD    = dd->makeDDFromMatrix(inputMatrix);
+
+    auto outputMatrix = dd->getMatrix(matDD);
+
+    for (std::vector<double>::size_type i = 0; i < outputMatrix.size(); i++) {
+        for (std::vector<double>::size_type j = 0; j < outputMatrix.size(); j++) {
+            EXPECT_EQ(inputMatrix[i][j], outputMatrix[i][j]);
+        }
+    }
```

Comment:
I would suspect that
```c++
EXPECT_EQ(inputMatrix, outputMatrix);
```
works just as well and is way simpler

Reply:
Done